File: app/src/main/java/com/task/paginationlist/data/utils/ImageLoader.java
package com.task.paginationlist.data.utils;

import android.widget.ImageView;

import com.squareup.picasso.Picasso;
import com.task.paginationlist.R;

/**
 * Created by kirill on 11.07.17.
 */

public class ImageLoader implements ILoader {

    private Picasso picasso;

    public ImageLoader(Picasso picasso) {
        this.picasso = picasso;
    }

    @Override
    public void loadImage(String url, ImageView imageView) {
        picasso.load(url).placeholder(R.drawable.image_placeholder).into(imageView);
    }
}

File: app/src/main/java/com/task/paginationlist/data/utils/ImageLoader.kt
package com.task.paginationlist.data.utils

import android.widget.ImageView

import com.squareup.picasso.Picasso
import com.task.paginationlist.R

/**
 * Created by kirill on 11.07.17.
 */

class ImageLoader(private val picasso: Picasso) : ILoader {

    override fun loadImage(url: String, imageView: ImageView) {
        picasso.load(url).placeholder(R.drawable.image_placeholder).into(imageView)
    }
}

